patch/peerlookup.patch

```diff
@@ -0,0 +1,12 @@
+--- peerlookup.c	2021-01-19 01:34:52.624027083 +0100
++++ peerlookup.c	2021-01-19 01:40:12.063429383 +0100
+@@ -7,6 +7,9 @@
+ #include "peer.h"
+ #include "noise.h"
+ 
++#define spin_lock_bh(lock) __raw_spin_lock_bh(&(lock)->rlock)
```

Comment:
Maybe we could use `#ifndef` to always patch these files? That way we don't have to do the version checks.

Reply:
The #ifndef is preprocessor directive and is used during compilation. Therefore, it cannot solve the problem of not exporting symbols from the kernel when loading a module.

Reply:
I see, I thought it was always a macro.